docs/public/content/vuln_db/manuals/kb_importer.md

```diff
@@ -0,0 +1,120 @@
+      "reason": "Manual review",
+      "affected": true
+    },
+    { 
+      "id": "pkg:maven/org.bouncycastle/bcprov-jdk15@1.43",
+      "reason": "Manual review",
+      "affected": false
+    }
+  ],
+  "notes": [
+    {
+      "links": [
+        "https://www.bouncycastle.org/releasenotes.html",
+        "https://ubuntu.com/security/notices/USN-3727-1"
+      ],
+      "text": "In Bouncy Castle JCE Provider version 1.55 and earlier the DSA does not fully validate ASN.1 encoding of signature on verification. It is possible to inject extra elements in the sequence making up the signature and still have it validate, which in some cases may allow the introduction of 'invisible' data into a signed structure."
+    },
+    {
+      "links": [
+        "https://lists.debian.org/debian-lts-announce/2018/07/msg00009.html"
+      ],
+      "text": "It was discovered that Bouncy Castle incorrectly handled certain crypto algorithms. A remote attacker could possibly use these issues to obtain sensitive information, including private keys."
+    }
+  ]
+}
+```
+
+Information about the fields of vulnerability `metadata.json` file:
+
+ * `vulnerability_id`: Vulnerability ID.
+ * `aliases`: Aliases used for the vulnerability id.
+ * `affected_artifacts`: Affected Libraries for the vulnerability.
+ * `notes`: This is an array of `links` and `text`. Here, the links array consists of links which provide comprehensive vulnerability information and the text is the vulnerability description. (optional, it must be provided for vulnerabilities not available from the NVD).
+
+
+Commit metadata.json example file:
+```json
+{
+  "repository" : "https://github.com/bcgit/bc-java",
+  "branch" : "DEFAULT_BRANCH",
+  "timestamp" : "1476490865",
+  "commit_id" : "843c2e60f67d71faf81d236f448ebbe56c62c647"
+}
+```
+
+Information about the fields of commit metadata.json file:
+
+ * `repository`: URL of the VCS repository hosting the library project.
+ * `branch`: Branch name.
+ * `timestamp`: Timestamp when the commit was made.
+ * `commit_id`: VCS commit id.
+
+ Example run for kb-importer:
+```sh
+java -Dvulas.shared.backend.header.X-Vulas-Client-Token=<token> -Dvulas.shared.cia.serviceUrl=<cia-url> -Dvulas.shared.backend.serviceUrl=<backend-url> -jar kb-importer-3.1.14-SNAPSHOT-jar-with-dependencies.jar -d ./CVEdata/CVE-2016-1000338 -v -u -o
+```
+
+## Usage with [Project KB](https://github.com/SAP/project-kb)
+
+Example workflow to export and import data from 2 different Eclipse Steady backend:
+
+1. Install Project-KB as given in [installing-the-kaybee-tool](https://github.com/SAP/project-kb#installing-the-kaybee-tool).
+2. Run `kaybee setup` to get the kaybeeconf.yaml file.
+3. Provide the backend URL in kaybeeconf.yaml for which you want to export the data.
+4. Run `kaybee import -c kaybeeconf.yaml -n 10` to import 10 Vulnerabilities from the specified backend URL. This command generates kaybee statements in `.kaybee/imported` folder.
```

Comment:
As discussed: separate one-time setup steps from the routine steps

Reply:
I have done these changes. Can you check if the new section with setup and 'Import data into Steady Backend' is fine?